dashboard/config/scripts_json/csa-self-paced-pl.script_json

```diff
@@ -36,7 +36,7 @@
   ],
   "lessons": [
     {
-      "key": "Navigating the Code.org learning platform",
+      "key": "Navigating the Codeorg learning platform",
```

Comment:
Will this also get messed up? Or is it only at the *end* of the key?

Reply:
This was also an issue. You can see Hannah linked to it in the slack thread